pkg/cli/environment.go

```diff
@@ -30,13 +30,17 @@ import (
 
 	"github.com/spf13/pflag"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
 
 	"helm.sh/helm/v3/pkg/helmpath"
 )
 
 // defaultMaxHistory sets the maximum number of releases to 0: unlimited
 const defaultMaxHistory = 10
 
+// defaultBurstLimit sets the default client-side throttling limit
+const defaultBurstLimit = 250
```

Comment:
Recommend setting the value to `100` as the default. The default in the Kubernetes rest client is `10` and `250` seems to be a dramatic change

Reply:
Fixed in https://github.com/helm/helm/pull/10842/commits/37608533d4f700b7281eff3e93f8dcf87ec5989b